posthog/clickhouse/batch_exports_log_entries.py

```diff
@@ -0,0 +1,68 @@
+from posthog.clickhouse.kafka_engine import KAFKA_COLUMNS, kafka_engine, ttl_period
+from posthog.clickhouse.table_engines import ReplacingMergeTree
+from posthog.kafka_client.topics import KAFKA_BATCH_EXPORTS_LOG_ENTRIES
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
+
+BATCH_EXPORTS_LOG_ENTRIES_TABLE = "batch_exports_log_entries"
+BATCH_EXPORTS_LOG_ENTRIES_TTL_WEEKS = 1
+
+BATCH_EXPORTS_LOG_ENTRIES_TABLE_BASE_SQL = """
+CREATE TABLE IF NOT EXISTS {table_name} ON CLUSTER '{cluster}'
+(
+    team_id UInt64,
+    batch_export_id VARCHAR,
+    run_id VARCHAR,
+    timestamp DateTime64(6, 'UTC'),
+    level VARCHAR,
```

Comment:
could consider enum and number for storage efficiency and typos avoidance

Reply:
Enum for level makes sense, I copied most of this from plugin_log_entries. Where do you say we should use number? Also for level? That would require mapping the numbers to their names for displaying to users.

Reply:
I've used this type as suggested 👍